### PR TITLE
Awesome Books refactored with Es6 Modules

### PR DESCRIPTION
**Pull request by Efiamotu-1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,53 +4,36 @@
...
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML
+- CSS
+- Javascript ES6
 
+## Live Demo
+https://efiamotu-1.github.io/Awesome-Books-With-ES6/
```
Comment (danmainah) [inline README.md:16]: - [ ] Kindly check your project, it is failing to load the add form  and list   both the demo and running it locally, ensure when a user clicks on add or list the page achieves the functionality, am get this page:

image

File: modules/date.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// import {DateTime} from '../node_modules/luxon/src/luxon.js'
```
Comment (danmainah) [inline modules/date.js]: - [ ] Kindly remove the commented code, it's not a good practice to have commented code in your codebase.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Efiamotu-1 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Efiamotu-1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**